+ Added Embedded Systems Topic

kamatajinkya opened the pull request:
+ Added Resources for Beginner and Intermediate Level
+ Added generals tips for embedded
+ Updated the list of topics

ajinkyakamat committed:
+ Added Embedded Systems Topic
+ Added Resources for Beginner and Intermediate Level
+ Added generals tips for embedded
+ Updated the list of topics

take2rohit commented on README.md:
```diff
@@ -22,6 +22,7 @@ Please click on following link to see the more info about the specific topic
 * [Control Theory](control-theory)
 * [Deep Learning](deep-learning)
+* [Embedded](embedded)
 * [Internet of Things-IoT](iot)
```
I think it should be `Embedded Systems`

akshaykulkarni07 replied:
it has to match the folder name, so `embedded-systems` specifically

take2rohit committed:
Removed hyphen

akshaykulkarni07 commented on embedded-systems/README.md:
```diff
@@ -0,0 +1,43 @@
+20 hours
+
+* [Tiva Launchpad Workshop](https://processors.wiki.ti.com/index.php/Getting_Started_with_the_TIVA%e2%84%a2_C-Series_TM4C123G_LaunchPad?DCMP=tivac&HQS=TM4C123G-Launchpad-Workshop)
+**_Why this works?:_**
```
Why this workshop? typo

kamatajinkya replied:
Its not a typo. Why this workshop as in "Why is this workshop useful?"

take2rohit reviewed (changes requested):
Please review these changes. Thanks

akshaykulkarni07 commented on embedded-systems/README.md:
```diff
@@ -0,0 +1,43 @@
+
+* [C Programming For Microcontrollers Featuring ATMEL's Butterfly and the Free WinAVR Complier](http://dsp-book.narod.ru/CPMicro.pdf)
+**_Why this book?:_**
+This book starts from the absolute basics. It presents C programming with an assumption that you are completely new to the programming scene, while simultenously ensuring a healthy level to technical recourse. It is a quick and easy read, with a lot of code and project examples. **_What to keep in min?:_**
```
what to keep in "mind" (typo)

akshaykulkarni07 reviewed (commented):
Some minor typos and inconsistencies (like micro controller v/s microcontroller and full stops not put every where consistently).

kamatajinkya commented:
Let me update this and push

aditya-shirwatkar commented:
I don't know anything about embedded systems, I think you've assigned the wrong reviewer :grimacing:

ajinkyakamat committed:
* Corrected mistakes in PR
* Changed name of topic to embedded systems
+ Added Hardware Design Section
+ Added Reference Design Archives Section
+ Added Forums, Trade... Section

ajinkyakamat committed:
Merge branch 'embedded' of github.com:IvLabs/resources into embedded

kamatajinkya commented:
> I don't know anything about embedded systems, I think you've assigned the wrong reviewer
@aditya-shirwatkar It not necessary that the reviewer must know about embedded systems. You can do a check on documentation rules and such

kamatajinkya commented:
Updated the document. If there is a spelling or a grammar mistake it would be nice if u guys can use the suggest changes feature to correct that.

akshaykulkarni07 committed:
minor grammar fixes

kamatajinkya commented:
@GlazeDonuts @take2rohit Need ur inputs to merge the branch
